sdk/eventhub/event-hubs/package.json

```diff
@@ -40,23 +40,23 @@
     "build:samples": "cd samples && tsc -p .",
     "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
     "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build:test": "npm run build:test:node",
+    "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
```

Comment:
Shouldn't this be `npm run build:test:node && npm run build:test:browser`?
I have taken Service Bus scripts as an example. I believe we run tests for both browser and node in CI for Service Bus

Reply:
The one in Service Bus will  need to be updated, they are equivalent for most part. The latter incurs `tsc` execution twice.

Reply:
understood. Since both ONLY_BROWSER & ONLY_NODE values are not set, both variations will be rolled up